genie-web/src/test/groovy/com/netflix/genie/web/agent/services/impl/AgentConnectionTrackingServiceImplSpec.groovy

```diff
@@ -0,0 +1,222 @@
+/*
+ *
+ *  Copyright 2020 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.agent.services.impl
+
+import com.netflix.genie.web.agent.services.AgentRoutingService
+import org.springframework.scheduling.TaskScheduler
+import spock.lang.Specification
+
+import java.time.Instant
+import java.util.concurrent.ScheduledFuture
+import java.util.function.Supplier
+
+class AgentConnectionTrackingServiceImplSpec extends Specification {
+
+    AgentConnectionTrackingServiceImpl service
+    AgentRoutingService agentRoutingService
+    TaskScheduler taskScheduler
+    Supplier<Instant> timeSupplier
+    long streamTTL = AgentConnectionTrackingServiceImpl.STREAM_EXPIRATION_PERIOD
```

Comment:
Why not just use the constant directly? Just shorter name?

Reply:
Inlined, it was used only in a couple of places after all